files/es/web/css/zoom/index.md

```diff
@@ -32,32 +27,30 @@ zoom: 0.7;
 /* Global values */
 zoom: inherit;
 zoom: initial;
+zoom: revert;
+zoom: revert-layer;
 zoom: unset;
 ```
 
-## Sintaxis
-
 ### Valores
 
 - `normal`
   - : Dibuja el elemento con su tamaño normal.
 - `reset` {{non-standard_inline}}
 
-  - : No cambia la escala del elemento (por ejemplo al presionar `<kbd>Ctrl</kbd>  \+  <kbd>-</kbd>` o `<kbd>Ctrl</kbd>  \+ <kbd>+</kbd>` )
-   
-  - si el usuario aplica `non-pinch-zooming`.
-    Sólo soportado por WebKit (y posiblemente Blink).
+  - : No cambiar la escala de este elemento si el usuario aplica zoom no basado en toques al documento (p. ej.: presionando los atajos de teclado <kbd>Ctrl</kbd> \- <kbd>-</kbd> o <kbd>Ctrl</kbd> \+ <kbd>+</kbd>). Sólo soportado por `WebKit` (y posiblemente `Blink`).
```

Comment:
typo (?)

```suggestion
  - : No cambia la escala de este elemento si el usuario aplica zoom no basado en toques al documento (p. ej.: presionando los atajos de teclado <kbd>Ctrl</kbd> \- <kbd>-</kbd> o <kbd>Ctrl</kbd> \+ <kbd>+</kbd>). Sólo soportado por `WebKit` (y posiblemente `Blink`).
```

Reply:
- `normal`
  - : Render this element at its normal size.
- `reset` {{non-standard_inline}}
  - : Do not (de)magnify this element if the user applies non-pinch-based zooming (...)
  
  Lo interpreté como infinitivo, no es una `typo`.

Reply:
Hola! @marcelozarate perdón la demora, ya veo, solo que la frase menciona que al hacer zoom via touch no "cambia", no se está empleando como verbo. Muchas gracias!